Sample Data/Custom/beSECURE_Audit_CL.json

```diff
@@ -0,0 +1,3 @@
+[

```

Comment:
This sample data is very lean - please cover examples of different types of audit logs for a richer sample data set. It enables community to build on top of your data connector  by having rich sample data with different combinations of types.